src/compiler/checker.ts

```diff
@@ -960,8 +964,15 @@ namespace ts {
             if (targetSymbol) {
                 const name = specifier.propertyName || specifier.name;
                 if (name.text) {

```

Comment:
See test below `JavaScriptModules20` that tests namespace imports - they now work with this change as written (didn't before).